TM1py/Utils/Utils.py

```diff
@@ -1321,3 +1322,59 @@ def init_poolmanager(self, *args, **kwargs):
         if self.socket_options is not None:
             kwargs["socket_options"] = self.socket_options
         super(HTTPAdapterWithSocketOptions, self).init_poolmanager(*args, **kwargs)
+
+def sync_two_dimensional_cube(tm1_s: TM1Service, tm1_t: TM1Service, cube_name: str,
+                                  skip_non_existing_elements=False, sync_rules=False):
+    """
+    Function looks at 2-dimensional cube at source TM1 service and synchronizes it at target TM1 service. For example for security cube, }Clients cube, }DimensionAttribute Cube.
+    :tm1_s: TM1Service of Source
+    :tm1_t: TM1Service of Target
+    :cube_name: str, name of cube to be synchronized
+    :skip_non_existing_elements: bool, if True only the leaves Element of the Source Cube get synchronized. If False all elements, for example Security Groups
+    :sync_rules: bool, if True rules of Source cube get synchronized to the Target Cube
+    """
+
+    query = MdxBuilder.from_cube(cube_name)
+    query = query.non_empty(axis=0)
+    for dimension_name in tm1_s.cubes.get_dimension_names(cube_name):
```

Comment:
For compliance with v12 we need to spread the MDX query out over both axes.
Please set one selection on rows and the other one on columns